initialization/serverSetup.js

```diff
@@ -10,7 +10,7 @@ const scheduledCronJobs = require('~/cron-jobs/scheduledCronJobs')
 
 const serverSetup = async (app) => {
   await databaseInitialization()
-  await checkUserExistence()
+  // await checkUserExistence()
```

Comment:
it seems like you forgot to uncomment or remove it

Reply:
i did it intentionally, coz with a clean DB it will cast an error, so this must be refactored, but this refactor is out of the scope of this task.